nixpkgs.nix

```diff
@@ -7,7 +7,7 @@
 # The SHA256 will be printed as the last line of stdout.
 
 import ./fetch-nixpkgs.nix {
```

Comment:
There's a newer version of the fetch-nixpkgs which doesn't require two hashes but that doesn't need to change in this PR.